_posts/2017-08-08-angular4-polymer2-bridge.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: "Angular + Polymer Bridge"
+date: 2017-08-08 20:14:09
+description: ¡Usa webcomponents de Polymer en tu app Angular!
+tags: Angular Polymer
+categories: Frontend Samples
+twitter_text: Angular + Polymer Bridge
+author: Aaron Rosas (aarr90@gmail.com)
```

Comment:
Great Article! Please, change the **author** value to just "aaron". It will be the key to your profile data